lib/records/RecCore.cc

```diff
@@ -197,7 +205,7 @@ RecCoreInit(RecModeT mode_type, Diags *_diags)
   g_num_records = 0;
 
   // initialize record array for our internal stats (this can be reallocated later)
-  g_records = (RecRecord *)ats_malloc(REC_MAX_RECORDS * sizeof(RecRecord));
+  //  g_records = (RecRecord *)ats_malloc(REC_MAX_RECORDS * sizeof(RecRecord));
```

Comment:
Why not just delete this line?

Reply:
[See here](https://github.com/apache/trafficserver/pull/4911#issuecomment-460090862).

Reply:
I should also note that while the comment claims it could be re-allocated, there's no code that does that and I don't think it would work if tried, because the hash table stores raw pointers. One might wonder why it was done this way, but I suspect an artifact of how TCL hash tables worked. Maybe I should make a PR to change the value type to `Record` instead of `Record*` and get rid of `g_records` entirely.